phys2bids/utils.py

```diff
@@ -280,3 +282,54 @@ def load_heuristic(heuristic):
         except Exception as exc:
             raise ImportError(f'Failed to import heuristic {heuristic}: {exc}')
     return mod
+
+
+def append_list_as_row(file_name, list_of_elem):
+    # Open file in append mode
+    with open(file_name, 'a+', newline='') as write_obj:
+        # Create a writer object from csv module
+        csv_writer = writer(write_obj, delimiter='\t')
+        # Add contents of list as last row in the csv file
+        csv_writer.writerow(list_of_elem)
+
+
+def participants_file(indir, outdir, yml, sub):
```

Comment:
I think this function should go in the bids.py script since is part of the bidsdifiy process

Reply:
Done. Thanks! I'll push those changes in a minute, once I fix the yaml dependency issue.